ext/ruby_newt/ruby_newt.c

```diff
@@ -1475,9 +1496,9 @@ void Init_ruby_newt(){
   rb_define_method(cCheckbox, "set_flags", rb_ext_Checkbox_SetFlags, -1);
 
   cRadioButton = rb_define_class_under(mNewt, "RadioButton", cWidget);
-  rb_define_singleton_method(cRadioButton, "new", rb_ext_RadioButton_new, 5);
-  rb_define_method(cRadioButton, "get", rb_ext_Checkbox_GetValue, 0);
-  rb_define_method(cRadioButton, "set", rb_ext_Checkbox_SetValue, 1);
```

Comment:
Is this dropped for no reason?

Reply:
I originally decided to have RadioButton inherit from Checkbox in order to implement these, but while trying them out I discovered that get() only returns '*' or ' ' depending on whether the Radiobutton is set or not. You can use set() to set a Radiobutton's value to any character, but it doesn't modify the return value of Radiobutton.get_current() while allowing multiple Radiobuttons to be displayed as selected.

I don't really believe this to be useful behavior, so I disabled it. I guess it should probably have been included in the 'nocompat' branch. I could add it back if you prefer.